edn/tests/tests.rs

```diff
@@ -53,6 +53,7 @@ fn test_nil() {
 
 #[test]
 fn test_nan() {
+    assert_eq!(nan("#fNaN").unwrap(), Float(OrderedFloat(f64::NAN)));
```

Comment:
(the reason I made the change in the first place was because # +Infinity and # -Infinity wasn't parseable).